scripts/leapfrog/post_leap.sh

```diff
@@ -19,12 +19,20 @@
 set -e -u -x
 set -o pipefail
 
+export RPCM_VARIABLES=${RPCM_VARIABLES:-/etc/openstack_deploy/user_rpcm_variables.yml}
+
 echo "POST LEAP STEPS"
+
+if [[ ! -f "${RPCM_VARIABLES}" ]]; then
+  cp "${RPCO_DEFAULT_FOLDER}/etc/openstack_deploy/user_rpcm_variables.yml" "${RPCM_VARIABLES}"
```

Comment:
cp line is wrong, and should probably be 👍 
``
cp "${RPCO_DEFAULT_FOLDER}/rpcd/etc/openstack_deploy/user_rpcm_variables.yml" "${RPCM_VARIABLES}"
``